lib/parliament/grom/decorator/answer.rb

```diff
@@ -0,0 +1,30 @@
+module Parliament
+  module Grom
+    module Decorator
+      # Decorator namespace for Grom::Node instances with type: https://id.parliament.uk/schema/Answer.
+      module Answer
+        include Helpers::DateHelper
+        # Alias answerText with fallback.
+        #
+        # @return [String, String] the text of the Grom::Node or an empty string.
+        def text
+          respond_to?(:answerText) ? answerText : ''
+        end
+
+        # Alias answerGivenDate with fallback.
+        #
+        # @return [DateTime, nil] the answer date of the Grom::Node or nil.
+        def answer_given_date
+          @answer_given_date ||= respond_to?(:answerGivenDate) ? DateTime.parse(answerGivenDate) : nil
+        end
+
+        # Alias answerHasAnsweringPerson with fallback.
+        #
+        # @return [Grom::Node, nil] the answering_person of the Grom::Node or nil.
+        def answering_person
+          respond_to?(:answerHasAnsweringPerson) ? answerHasAnsweringPerson.first : nil
```

Comment:
Is there a situation where multiple can answer a question? Like if someone interrupts an answer to chip something in? (does that ever happen)

Reply:
Nope- answers are only ever attributed to one person. A department might put together an answer to a question but it will only ever be answered by one person.

Reply:
👍